### PR TITLE
Chk 1293 add notify retry queue

### PR DESCRIPTION
**Pull request by pietro-tota**
### List of changes
Add ecommerce notification-retry-queue where will be enqueued events to trigger user receipt notification retry

### Motivation and context
Add new queue that will be consumed by ecommerce event dispatcher for retry notification 

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-common/03_storage.tf [hunk 1]
```diff
@@ -104,6 +104,11 @@ resource "azurerm_storage_queue" "transactions_refund_queue" {
   storage_account_name = module.ecommerce_storage.name
 }
 
+resource "azurerm_storage_queue" "transactions_notification_retry_queue" {
```
Comment (giovanniberti) [inline src/domains/ecommerce-common/03_storage.tf:107]: 🚲 

```suggestion
resource "azurerm_storage_queue" "transactions_notifications_retry_queue" {
```

File: src/domains/ecommerce-common/03_storage.tf [hunk 2]
```diff
@@ -104,6 +104,11 @@ resource "azurerm_storage_queue" "transactions_refund_queue" {
...
+  name                 = "${local.project}-transaction-notification-retry-queue"
```
Comment (giovanniberti) [inline src/domains/ecommerce-common/03_storage.tf:108]: 🚲 
```suggestion
  name                 = "${local.project}-transaction-notifications-retry-queue"
```

**Review by giovanniberti — DISMISSED** [review]

File: src/domains/ecommerce-common/03_storage.tf (hunk 1 above)
Comment (pietro-tota) [reply src/domains/ecommerce-common/03_storage.tf]: Fixed with https://github.com/pagopa/pagopa-infra/pull/819/commits/4a602745b4b8fd47031ee88b4b684b69a4667ddd

File: src/domains/ecommerce-common/03_storage.tf (hunk 2 above)
Comment (pietro-tota) [reply src/domains/ecommerce-common/03_storage.tf]: Fixed with https://github.com/pagopa/pagopa-infra/pull/819/commits/4a602745b4b8fd47031ee88b4b684b69a4667ddd

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**